app/Resources/views/full/blog.html.twig

```diff
@@ -37,15 +38,15 @@
                                     </div>
                                 </div>
                             </div>
-                            <hr class="space-top" />
+                            <hr class="space-top">
                         </div>
                     </div>
                 </div>
 
                 <div class="container"><div class="row"><div class="col-md-10 col-md-offset-1 space-bottom"><div class="row">
             {% else %}
                 <div class="{% if loop.index % 2 == 1 %}col-md-offset-2 {% endif %}col-md-5 space-top">
-                    {{ render_esi(controller('ez_content:viewLocation', {
+                    {{ render_esi(controller('ez_content:viewContent', {
                         locationId: post.contentInfo.mainLocationId,
```

Comment:
Should I always use both `locationId` and `contentId` when it's possible? documentation doesn't say nothing about this approach.